lib/federation/initialize.py

```diff
@@ -41,22 +41,22 @@ def main():
     headers["Authorization"] = f"Bearer {token}"
 
     print("Adding servers to federation...")
-    url = f"{get_env_value('FEDERATION_SERVICE_URL')}/federation/v1/servers"
+    url = f"http://localhost:{get_env_value("FEDERATION_PORT")}/federation/v1/servers"
     response = requests.request("POST", url, headers=headers, json=server)
     # add other federated servers here
     if response.status_code != 200:
         print(f"POST response: {response.status_code} {response.text}")
-    url = f"{get_env_value('FEDERATION_SERVICE_URL')}/federation/v1/servers"
+    url = f"http://localhost:4232/federation/v1/servers"
```

Comment:
What is this bit for?

Reply:
FEDERATION_SERVICE_URL is not resolvable on our prod instance (candig.uhnresearch.ca exposes only port 80, and proxies requests to port 5080). Instead, I figured localhost would always resolve properly. Come to think of it though, I could use FEDERATION_PORT instead of hardcoding 4232.